src/views/address/confirm.html

```diff
@@ -34,10 +35,16 @@ <h3>{{translate("pages.address-confirm.previous")}}</h3>
 
   {% endif %}
 
-  {{hmpoSubmit(ctx, {
-    id: "addPreviousAddresses",
-    key: "Add another address path"
-  })}}
+  {% call hmpoForm(ctx) %}
+    {{ hmpoSubmit(ctx, {
+        text: translate("buttons.next"),
+        attributes: {
+        "data-id": "next"
```

Comment:
It's annoying that in order to add a `data-id` property to the button we can't use the automatic layout.

If we hit this again when using the form inputs directly, I think we should do a small spike and see if we can contribute some `data-id` enable/disable functionality back up into components.